src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(User::getAge)
+                .filter(age -> 30 <= age && age <= 45)
```

Comment:
```suggestion
                .filter(User::범위검증?())
```

메소드로 추출할필요는 없지만, 
코드의 읽는 흐름으로 보았을때, 메소드 참조로 한다면 가독성에 좀 더 좋지 않을까? 생각이되네요! ㅎ